aircmd/models/github.py

```diff
@@ -0,0 +1,49 @@
+from functools import wraps
+from typing import Any, Callable
+
+from prefect.context import FlowRunContext
+
+from ..actions.githubactions import github_status_update_hook
+from ..models.settings import GlobalSettings
+
+
+def github_integration(func: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
I tried to think of other, better ways to do this rather than Yet Another Decorator but this really ended up being the cleanest solution